packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -269,7 +269,7 @@ export const getCanCreateChat = createSelector(
         action = ChatPermissionAction.NONE
       } else if (
         userPermissions.permits === ChatPermission.FOLLOWEES &&
-        !user?.does_current_user_follow
+        !user?.does_follow_current_user
```

Comment:
🤔 wow how did this go unseen for so long

Reply:
i lol'd when i found it